README.md

```diff
@@ -21,10 +21,37 @@ To run, `pip install vinyldns-python` and then:
 >>> local_client.list_zones()
 ```
 
+## Contributing
+
+**Requirements**
+
+* `python3`
+* `pip`
+* `virtualenv`
+
+To get started, you will want to setup your virtual environment.  Ensure that you have `virtualenv`
+installed `>pip install virtualenv`.  Then, simply run `./bootstrap.sh` to setup your environment (unless
+you really want all these dependencies to be installed locally, which we do not recommend).
+
+Activate your virtual environment `>.virtualenv/bin/activate` and you will be ready to go!
+
+**Unit Tests**
+Unit tests are developed using [pytest](https://docs.pytest.org/en/latest/).  We use
+[Responses](https://github.com/getsentry/responses), which allows for simple mocking of HTTP endpoints.
+
+To run unit tests, you can simply run `python3 setup.py test`.  To target a specific test, you can
+run `python3 setup.pyt test -a "-k my_test"`
+
+**Running a full build**
+When you are finished writing your code, you will want to run everything, including linters.  The
+simplest way to do this is to run `tox -e check,py36`, which will run static checks and run unit tests
```

Comment:
Might be useful to link to [tox docs](https://tox.readthedocs.io/en/latest/) or indicate `pip install tox` can be used to install it.